shared/actions/fs/platform-specific.desktop.tsx

```diff
@@ -90,11 +90,8 @@ const _openPathInSystemFileManagerPromise = (openPath: string, isFolder: boolean
   new Promise((resolve, reject) => {
     if (isFolder) {
       if (isWindows) {
-        if (Electron.remote.shell.openItem(openPath)) {
-          resolve()
-        } else {
-          reject(new Error('unable to open item'))
-        }
+        Electron.remote.shell.openPath(openPath)
+        resolve()
```

Comment:
No idea if this is right. Relevant docs at https://github.com/electron/governance/blob/master/wg-api/spec-documents/shell-openitem.md

Reply:
should be
```
const error = await Electron.remote.shell.openPath(openPath);
if (error) {
reject(new Error('unable to open item'))
} else {
resolve()
}
```

Reply:
I'm getting this

```
actions/fs/platform-specific.desktop.tsx:93:23 - error TS1308: 'await' expressions are only allowed within async functions and at the top levels of modules.

93         const error = await Electron.remote.shell.openPath(openPath);
                         ~~~~~

  actions/fs/platform-specific.desktop.tsx:90:15
    90   new Promise((resolve, reject) => {
                     ~~~~~~~~~~~~~~~~~~~~~~
    Did you mean to mark this function as 'async'?


Found 1 error.

error Command failed with exit code 2.
```

should I make the handler async?

Reply:
you can do that or use `then`
```
Electron.remote.shell.openItem(openPath).then(error => {
if (error) {
reject(new Error('unable to open item'))
} else {
resolve()
}
})
```

Reply:
great, seems to have worked!